lib/scarpe/edit_box.rb

```diff
@@ -0,0 +1,52 @@
+module Scarpe
+  class EditBox
+    attr_reader :text, :height, :width
+
+    alias_method :function_name, :object_id
+
+    def initialize(app, text = nil, height: nil, width: nil, &block)
+      @app = app
+      @text = text || block.call
```

Comment:
This deviates from `edit_line` slightly. I thought it better to allow the initial block be an option for setting content since there's a possibility of the content being quite large. This way you can do this:
```ruby
edit_box(width: "100%") do
  "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim."
end
```
instead of being stuck with
```ruby
edit_box("Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim.", width: "100%")
```

I'm happy to revert though if I'm way off base here